### DISCUSSION TITLE
The global audio offset suggestion will adjust the value inversely

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#26283] [Q&A] [answered]
For example, if there is a value of +100ms, it is recommended to reduce it by 90ms. However, clicking the apply suggested button will adjust the offset to -90ms instead of +10ms.
I'm not sure if that's how he works.
Reports from other users: https://osu.ppy.sh/comments/2990475

**Comment by peppy** [comment]
We're working on a fix for this.

**Comment by Jan-Arbus** [comment, marked as answer]
https://github.com/ppy/osu/pull/26267
